tests/nobiru/test_nobiru_list.py

```diff
@@ -65,3 +65,23 @@ def test_delete_with_non_existant_entity(self):
             err.exconly()
             == "werkzeug.exceptions.NotFound: 404 Not Found: DummyRelation not found"
         )
+
+    def test_find(self):
+        fake = self.create_fake_model()
+        value_id = fake.values[0].id
```

Comment:
I'm looking at this, and I forgot I had wrote these helpers. Just thinking out loud here. I think values is kind of a confusing name. I think children would be more clear. Because it would be the associated children of the fake model. I think values is not clear what its doing unless you have all the context in your head. What do you think?

Reply:
Sorry  I didn't get back on this, I've been distracted by the new job for the last couple of weeks.  Did you still want to change this to children?

Reply:
No worries, I'm more interested to know if you would think that wording would make more sense or not. I'm ok to leave it as is for now unless you have strong opinion about changing the wording